src/app/public/pages/login/login.component.html

```diff
@@ -0,0 +1,85 @@
+<div class="window">
+  <div class="window__wrap">
+
+    <div class="title">
+      <span>entrance in  <br> marking board</span>
+      <span class="title-red">.</span>
+    </div>
+
+
+    <form class="form" [formGroup]="form" (ngSubmit)="submit()">
+      <div
+        class="form__control"
+        [ngClass]="{invalid: form.controls.email.touched && form.controls.email.errors}"
+      >
+        <input type="email" placeholder="Email" formControlName="email" value="test@gmail.com">
+        <div
+          *ngIf="form.controls.email.touched && form.controls.email.invalid"
+          class="validation">
+          <small *ngIf="form.controls.email.hasError('required')">
+            Field email is required
+          </small>
+          <small *ngIf="form.controls.email.hasError('email')">
+            Field must be type email
+          </small>
+        </div>
+      </div>
+
+      <div
+        class="form__control mb-24"
+        [ngClass]="{invalid: form.controls.password.touched && form.controls.password.errors}"
+      >
+        <input type="password" placeholder="Password" formControlName="password" value="123456789Zaq">
+        <div
+          *ngIf="form.controls.password.touched && form.controls.password.invalid"
+          class="validation">
+          <small *ngIf="form.controls.password.hasError('required')">
+            Field password is required
+          </small>
+          <small *ngIf="form.controls.password.hasError('minlength')">
+            Password must be more than
+            {{form.controls.password.errors?.['minlength'].requaredLength}} characters.
+            Now is
+            {{form.controls.password.errors?.['minlength'].actualLength}}
+          </small>
+          <small *ngIf="form.controls.password.hasError('pattern')">
+             Password must contain at least one uppercase letter, one lowercase letter, and one digit
+          </small>
+        </div>
+      </div>
+
+
+      <label class="form__control mb-32 form__control-checkbox">
+        <input type="checkbox"
+               [(ngModel)]="isChecked"
```

Comment:
You can't use form and ngModel in the same time, just add new form control to your form